app/src/organisms/Breadcrumbs/index.tsx

```diff
@@ -46,7 +46,7 @@ function CrumbName({ crumbName, isLastCrumb }: CrumbNameProps): JSX.Element {
       ? getProtocolDisplayName(
           storedProtocol.protocolKey,
           storedProtocol.srcFileNames,
```

Comment:
Perhaps we should add the optional chaining here if it might be helpful on l49?

```suggestion
          storedProtocol?.protocolKey,
          storedProtocol?.srcFileNames,
```

Reply:
Good catch, actually all of the optional chains here are unnecessary. I updated the type for storedProtocol and mostRecentAnalysis will be coalesced to `null` upstream of this point (which this get display name helper function is prepared for as well)